common/supplemental/coverageLevels.xml

```diff
@@ -444,6 +444,11 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<coverageLevel	value="moderate"	inLanguage="%devaDefaultLanguages"	match="numbers/symbols[@numberSystem='deva']/minusSign"/>
 		<coverageLevel	value="moderate"	inLanguage="%devaDefaultLanguages"	match="numbers/symbols[@numberSystem='deva']/percentSign"/>
 		<coverageLevel	value="moderate"	inLanguage="%devaDefaultLanguages"	match="numbers/symbols[@numberSystem='deva']/plusSign"/>
+		<coverageLevel	value="moderate"	inLanguage="%khmrDefaultLanguages"	match="numbers/symbols[@numberSystem='khmr']/decimal"/>
+		<coverageLevel	value="moderate"	inLanguage="%khmrDefaultLanguages"	match="numbers/symbols[@numberSystem='khmr']/group"/>
+		<coverageLevel	value="moderate"	inLanguage="%khmrDefaultLanguages"	match="numbers/symbols[@numberSystem='khmr']/minusSign"/>
+		<coverageLevel	value="moderate"	inLanguage="%khmrDefaultLanguages"	match="numbers/symbols[@numberSystem='khmr']/percentSign"/>
+		<coverageLevel	value="moderate"	inLanguage="%khmrDefaultLanguages"	match="numbers/symbols[@numberSystem='khmr']/plusSign"/>
```

Comment:
yeah, this is a problem (but not this ticket's problem). There are many missing scripts here. I filed another ticket somewhere.

Reply:
seems this should be 100% automatic

Reply:
Ticket for identifying and fixing scripts with missing coverage is: https://unicode-org.atlassian.net/browse/CLDR-16865